cli/flags.go

```diff
@@ -186,6 +186,16 @@ var ioFlags = []cli.Flag{
 		Value: string(hostSelectTypeWeighed),
 		Usage: fmt.Sprintf("Host selection algorithm. Can be %q or %q", hostSelectTypeWeighed, hostSelectTypeRoundrobin),
 	},
+	cli.BoolFlag{
+		Name:   "resolve-host",
+		Usage:  "Resolve the host(s) ip(s) (including multiple A/AAAA records). This can break SSL certificates, use --insecure if so",
+		Hidden: true,
+	},
+	cli.BoolFlag{
+		Name:   "shuffle-hosts",
```

Comment:
I don't really see the point of this. Shuffling shouldn't really have any practical implications.

I would rather that it is semi-repeatable. IMO the option should just be removed.

Reply:
I removed the shuffle-hosts option

sorry for the long long delay of fixing :)